packages/babel-helper-module-transforms/src/rewrite-this.js

```diff
@@ -12,10 +13,11 @@ const rewriteThisVisitor = {
     path.replaceWith(path.scope.buildUndefinedNode());
   },
   Function(path) {
-    if (!path.isArrowFunctionExpression()) path.skip();
+    if (path.isMethod()) skipAllButComputedKey(path);
+    if (!path.isArrowFunctionExpression() && !path.isMethod()) path.skip();
```

Comment:
Nit: you can use `else if` and avoid checking `isMethod` again.

Reply:
@nicolo-ribaudo Made the changes. PTAL now.